Root/OptimizationDump.cxx

```diff
@@ -377,13 +377,17 @@ EL::StatusCode OptimizationDump :: execute ()
   }
 
   if(!m_inputMET.empty() && !m_inputJets.empty()){
-    m_effectiveMass = VD::Meff_inclusive(in_met, in_jets, in_muons, in_electrons)/1000.;
+    
+    ConstDataVector<xAOD::JetContainer> presel_jets = VD::subset_using_decor(in_jets, VD::decor_vd_pass_Presel, 1);
+    //std::cout << in_jets->size() << " " << presel_jets.size() << std::endl;
+    m_effectiveMass = VD::Meff_inclusive(in_met, presel_jets.asDataVector(), in_muons, in_electrons)/1000.;
```

Comment:
Yeah, so that is potentially annoying. There would be two classes: "signal_for_meff_and_stuff" and "signal_for_njets." When the pT cut changes in 1L, they just change the definition of "signal_for_njets" while leaving the former untouched. For us, we don't really care at the moment, and can just leave this as the same thing.